app/javascript/src/components/Dashboard/Notes/NoteItem/NoteItem.jsx

```diff
@@ -44,9 +45,9 @@ const NoteItem = ({
         />
         <div className="flex items-center">
           <Clock color="#1e1e20" size={22} />
-          <Tooltip content="Wednesday, 10:30AM" position="bottom">
+          <Tooltip content={formatDate(createdAt)} position="bottom-end">
```

Comment:
`formatDate` is a very generic util name. We can format the date into any particular format in the world. But what the format is, that's not conveyed here. Thus either pass in the format as a named argument or update the util name to convey the same.